### PR TITLE
CO-9690 Add more logging around Sand tokens

### PR DESCRIPTION
**Pull request by johnwu96822**
More logging to find out what's going on with Sand auth.

- [x] @johnny-lai

### CODE REVIEW

**Comment by johnny-lai** [comment]
lgtm

File: src/main/java/com/coupa/kafka/security/sasl/oauth/KafkaTokenValidator.java [hunk 1]
```diff
@@ -73,16 +71,19 @@ private void validateToken(OAuthBearerValidatorCallback callback) throws IOExcep
 
     AllowedResponse resp;
     try {
+      LOGGER.info("Verifying token: " + token.substring(0, 4));
```
Comment (johnny-lai) [inline src/main/java/com/coupa/kafka/security/sasl/oauth/KafkaTokenValidator.java]: I think this log is not necessary

File: src/main/java/com/coupa/kafka/security/sasl/oauth/KafkaTokenValidator.java [hunk 2]
```diff
@@ -73,16 +71,19 @@ private void validateToken(OAuthBearerValidatorCallback callback) throws IOExcep
...
+      LOGGER.error("Error verifying Sand token: " + e.getMessage());
       throw new IOException(e.getMessage(), e);
     }
     SandOAuthToken tk = new SandOAuthToken(token, resp);
-    LOGGER.debug("SandOAuthToken: " + tk);
-    if (!resp.isAllowed()) {
-      OAuthBearerValidationResult.newFailure("Token access denied");
-      LOGGER.debug("Access denied to token: " + token);
+    if (resp.isAllowed()) {
+      LOGGER.info("Sand access granted for: " + tk.principalName());
```
Comment (johnny-lai) [inline src/main/java/com/coupa/kafka/security/sasl/oauth/KafkaTokenValidator.java]: Let's do `debug` for allowed. This is what the SimpleAclAuthorizer does.

(unless you are still debugging)

**Comment by johnny-lai** [comment]
lgtm

**Comment by johnwu96822** [comment]
[lockit v4.9.2] merged via Lockit CLI on 2019-10-11 16:28:05.85421 -0700 PDT m=+5.331311940

**Pull request merged**